catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswSourceConfiguration.java

```diff
@@ -92,7 +103,11 @@ public String getPassword() {
     }
 
     public void setPassword(String password) {
```

Comment:
Added the StringUtils.isNotEmpty(password) check along with keeping the null check of the encryption service.